x-pack/plugins/actions/server/plugin.ts

```diff
@@ -232,13 +234,19 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
       preconfiguredActions,
     } = this;
 
-    const encryptedSavedObjectsClient = plugins.encryptedSavedObjects.getClient();
+    const encryptedSavedObjectsClient = plugins.encryptedSavedObjects.getClient({
+      includedHiddenTypes,
+    });
+    const getScopedSavedObjectsClient = (request: KibanaRequest) =>
```

Comment:
I believe this SO client is the one that we pass to action executor functions - do we want to specifically include the hidden types here?  I'm not sure if this client is also used by the framework (ie, not just in the action executor), in which case it would, or we'd need separate ones.